packages/orbit-components/.storybook/orbitDecorator.tsx

```diff
@@ -38,7 +38,7 @@ const OrbitDecorator = (storyFn, context) => {
         </Text>
         {children}
         <div style={{ marginTop: 20 }} dir="ltr">
-          <Source code={jsxToString(children, options)} language="jsx" format={false} />
+          {/*<Source code={jsxToString(children, options)} language="jsx" format={false} />*/}
```

Comment:
There is an issue with this Source component. I tried to use this workaround but I wasn't able to make it work :/ https://github.com/storybookjs/storybook/issues/22088#issuecomment-2076906244

The issue started with v8, in 7.6.17 everything was working normally.

<img width="1102" alt="Snímek obrazovky 2024-07-05 v 12 12 46" src="https://github.com/kiwicom/orbit/assets/43321271/096056c3-5ff9-4308-a8c8-b48e72816955">